src/Engine/Renderer/Displayable/DisplayableObject.hpp

```diff
@@ -40,17 +42,19 @@ class RA_ENGINE_API Displayable
 
     /// Returns the underlying AbstractGeometry, which is in fact a TriangleMesh
     /// \see getTriangleMesh
-    virtual const Core::Geometry::AbstractGeometry& getGeometry() const = 0;
-    virtual Core::Geometry::AbstractGeometry& getGeometry()             = 0;
+    /// \todo not sure the displayable must have an abstract geometry ...
```

Comment:
what about this todo?

Reply:
Well it's a todo to be discussed during RadiumEngine meeting one day